docs/geedocs/5.3.1/answer/176738.html

```diff
@@ -258,6 +258,8 @@ <h2 id="cache_configuration">Cache configuration</h2>
 
 <p>Within <code>misc.xml</code>, the following options are deprecated and are treated as if they are permanently enabled: <code>DisablePacketLevelVersionCachePurge</code> and <code>ConsolidateListenerNotifications</code>.</p>
 
+<p>Also included within <code>misc.xml</code> is an experimental option, which allows the user to limit the caches by memory usage instead of number of objects. This option is controlled by <code>LimitMemoryUtilization</code> and its default value is false. The maximum memory usage for each cache can be set with <code>MaxAssetCacheMemorySize</code> and <code>MaxVersionCacheMemorySize</code>. This size is measured in bytes and the default setting is 1GB. While this option is enabled, the caches will not be limited by the number of objects in the cache.</p>
```

Comment:
Probably ought to add a note here that the estimate of cache memory usage is very low right now, so this is not recommended for production environments.

Also, it would be easier to read if you broke the text over multiple lines.